radio/src/gui/colorlcd/widgets/modelbmp.cpp

```diff
@@ -34,6 +34,20 @@ class ModelBitmapWidget: public Widget
 
     void refresh(BitmapBuffer * dc) override
     {
+      std::string filename = std::string(g_model.header.bitmap);
+
+      // set font colour from options[0]
+      lcdSetColor(persistentData->options[0].value.unsignedValue);
```

Comment:
You should never have to use this! This is modifying the colour palette, which is not needed at all in C++, as you can pass colours (RGB565) directly.

Reply:
Looks like a few more widgets need re-writing then... as this is how the text colour was set in both the value and text widgets :laughing:

Reply:
😬 My best wishes! You just got yourself a job! 🥳

Reply:
Well, in that case, here's my two weeks notice, effective 15 days ago! :-P /jk